nepi/settings_production.py

```diff
@@ -31,6 +31,8 @@
 if 'migrate' not in sys.argv:
     INSTALLED_APPS.append('raven.contrib.django.raven_compat')
 
+LOCALE_PATHS = ('/var/www/nepi/meaningfulconsent/locale',)
```

Comment:
We probably ought to be better about doing relative paths for these kinds of settings  anyway. Ie, `os.path.join(os.path.dirname(__file__), 'locale')`